packages/stark-ui/src/modules/table/components/column.component.ts

```diff
@@ -214,6 +214,40 @@ export class StarkTableColumnComponent extends AbstractStarkUiComponent implemen
 	@Input()
 	public stickyEnd = false;
 
+	/**
+	 * A function to generate tooltip for cells based on the value, its row and the name of the column.
+	 * Or a static string with the tooltip.
+	 */
+	@Input()
+	public cellTooltip?: ((value: any, row?: object, columnName?: string) => string) | string;
+
+	/**
+	 * Position where the tooltip should be displayed.
+	 * Default: "below"
+	 * Possible positions: above, below, left, right, before, after
+	 */
+	@Input()
+	public get cellTooltipPosition(): string {
+		return this._cellTooltipPosition;
+	}
+
+	public set cellTooltipPosition(tooltip: string) {
```

Comment:
Same remark about using specific types instead of just `string`. You could re-use the ones from the ColumnProperties :wink: